### PR TITLE
Feat: Updating CCAI Block with https://github.com/GoogleCloudPlatform/gcc-conversational_ai_analytics

### PR DESCRIPTION
**Pull request by Alex8ab**
# CHANGELOG - CCAI Insights block v2.0
v2.0 (2024-12-12)

## NEW
+ Compatible with BQ Schema v4 and higher
+ Added Agent Operations dashboard 
+ Added _agent_filter view 

## CHANGES
+ Refined insights model (adapted for BQ schema v4 and higer, more descriptive fields and functions)
+ BQ schema v4 and higher incorporate more data (Export your data to BigQuery for custom analysis or visualization in Looker)

### CODE REVIEW

File: Insights_looker_block/README.md
Comment (minhaz) [inline Insights_looker_block/README.md:1]: Why do we have two separate READMEs? can we fold into one in the root folder? I'm referring to the READMEs on the block/dashboards.

**Review by heyedher — APPROVED** [review]
LookML code looks with correct syntax

**Review by Itzaprado — APPROVED** [review]

File: Insights_looker_block/README.md
Comment (Alex8ab) [reply Insights_looker_block/README.md:1]: Well, we unified the READMEs for better explanation.

File: infra-as-code/environments/dev/backend.tf [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Google LLC
```
Comment (minhaz) [inline infra-as-code/environments/dev/backend.tf:1]: 2024?

File: infra-as-code/environments/dev/backend.tf
Comment (minhaz) [inline infra-as-code/environments/dev/backend.tf:1]: what is the purpose of the entire infra-as-code and the utils folder? how does it relate to the block? Shouldn't this be left out of this report?

Comment (Itzaprado) [reply infra-as-code/environments/dev/backend.tf:1]: Folders were removed

**Review by minhaz — APPROVED** [review]

**Pull request merged**